README.markdown

```diff
@@ -309,62 +310,116 @@ Manages a setting within an INI file.
+*Optional.* 
+
+Specifies a string to use between each setting name and value (e.g., to determine whether the separator includes whitespace). 
+
+Valid options: a string. 
+
+Default value: ' = '.
 
 ##### `name`
 
-*Optional.* Specifies an arbitrary name to identify the resource. Valid options: a string. Default value: the title of your declared resource.
+*Optional.* 
+
+Specifies an arbitrary name to identify the resource. 
+
+Valid options: a string. 
+
+Default value: the title of your declared resource.
 
 ##### `path`
 
-*Required.* Specifies an INI file containing the setting to manage. Valid options: a string containing an absolute path.
+*Required.* 
+
+Specifies an INI file containing the setting to manage. 
+
+Valid options: a string containing an absolute path.
 
 ##### `section`
 
-*Optional.* Designates a section of the specified INI file containing the setting to manage. To manage a global setting (at the beginning of the file, before any named sections) enter "". Defaults to "". Valid options: a string.
+*Optional.* 
+
+Designates a section of the specified INI file containing the setting to manage. To manage a global setting (at the beginning of the file, before any named sections) enter "". Defaults to "". 
+
+Valid options: a string.
 
 ##### `setting`
 
-*Required.* Designates a setting to manage within the specified INI file and section. Valid options: a string.
+*Required.* 
+
+Designates a setting to manage within the specified INI file and section. 
+
+Valid options: a string.
 
 ##### `show_diff`
 
-*Optional.* Prevents outputting actual values to the logfile. Useful for handling of passwords and other sensitive information. Possible values are:
+*Optional.* 
+
+Prevents outputting actual values to the logfile. Useful for the handling of passwords and other sensitive information. Possible values are:
   * `true`: This allows all values to be passed to logfiles. (default)
   * `false`: The values in the logfiles will be replaced with `[redacted sensitive information]`.
   * `md5`: The values in the logfiles will be replaced with their md5 hash.
 
-Global show_diff configuraton takes priority over this one -
+Global `show_diff` configuraton takes priority over this one:
 [https://docs.puppetlabs.com/references/latest/configuration.html#showdiff]([https://docs.puppetlabs.com/references/latest/configuration.html#showdiff].
-). Default value: 'true'.
+). Default value: `true`.
 
 ##### `value`
 
-*Optional.* Supplies a value for the specified setting. Valid options: a string. Default value: undefined.
+*Optional.* 
+
+Supplies a value for the specified setting. 
+
+Valid options: a string. 
+
+Default value: `undef`.
 
 ##### `section_prefix`
 
-*Optional.*  Designates the string that will appear before the section's name.  Default value: "["
+*Optional.*  
+
+Designates the string that will appear before the section's name.  
```

Comment:
Let's change "that will" to "to". And then same thing with the few other parameters that read like this.